index.js

```diff
@@ -1,34 +1,9 @@
-const fs = require('fs');
-const sampleSize = require('lodash.samplesize');
+const data = require('./generated/words.json');
 
-// Load the words into memory:
-const getWords = (fileName) => {
-  const fileContents = fs.readFileSync(__dirname + `/words/${fileName}`, {encoding:'ascii'})
-  return fileContents.split("\n");
-}
-const OBJECTS = getWords('objects.txt');
-const PREDICATES = getWords('predicates.txt');
-const TEAMS = getWords('teams.txt');
+const OBJECTS = data.objects;
```

Comment:
why indirect through the consts?  seems like we could export the items directly (`exports.objects = data.objects`)

( I think we could also directly export the data object (`exports = data`), although I think it's kind of nice to keep the exports visible in the file here, and it'll also make it easier to add utility functions later if we decided to.)

Reply:
I had tried `exports = data` yesterday, and couldn't then access the different lists for the example webpage. Not sure what I was missing/messing up.

Agreed on direct export.